framework/test/bench/BenchTest.cpp

```diff
@@ -0,0 +1,119 @@
+#include "benchmark/benchmark.h"

```

Comment:
I'd suggest naming this source file something more specific to what its meant to benchmark.